scripts/install.ps1

```
@@ -0,0 +1,617 @@
+    if ($Status -eq $null -or $Status.command -eq $null) { return $false }
+    $command = @($Status.command)
+    if ($command.Count -eq 0) { return $false }
+    try {
+        $registered = Normalize-PathDirectory ([string]$command[0])
+        $target = Normalize-PathDirectory $BuilderPath
+        return [string]::Equals($registered, $target, [System.StringComparison]::OrdinalIgnoreCase)
+    } catch { return $false }
+}
+function Stop-ServiceForUpdate([string]$BuilderPath) {
+    $status = Get-ServiceStatusForUpdate $BuilderPath
+    if ($status -eq $null -or -not [bool]$status.installed) { return }
```

PSScriptAnalyzer (1.25.0)

[warning] $null should be on the left side of equality comparisons.

Suggested fix: Use $null on the left hand side for safe comparison with $null.

(PSPossibleIncorrectComparisonWithNull)